tests/test_vectorizer.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from gensim.models import KeyedVectors
+from gensim.scripts.glove2word2vec import glove2word2vec
+
+from extra_model._vectorizer import Vectorizer
+
+
+@pytest.fixture()
+def vec():
+    # preprocess plain-text test embeddings to proper binary format for vectorizer
+    glove_file = "tests/resources/test_adjectives.vec"
+    tmp_file = "tests/resources/test_adjectives.tmp"
+    _ = glove2word2vec(glove_file, tmp_file)
+    model = KeyedVectors.load_word2vec_format(tmp_file)
+    model.save("tests/resources/test_adjectives.prepro")
+
+    return Vectorizer("tests/resources/test_adjectives.prepro")
+
+
+def test__word_exists(vec):
+    res = vec.get_vector("little")
```

Comment:
we should expose more info about the testing data to the end user so they understand the contents of the preprocessed data to assert the tests actually pass, or to add more tests etc

Reply:
what do you think is a good way to do it? This file right now is literally 4 lines - https://github.com/wayfair-incubator/extra-model/blob/main/tests/resources/test_adjectives.vec, should I just add a small comment with words that are in it?